packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2564,7 +2564,9 @@ export interface VideoGalleryProps {
 // @public
 export interface VideoGalleryRemoteParticipant extends VideoGalleryParticipant {
     isSpeaking?: boolean;
+    kind?: 'communicationUser' | 'phoneNumber' | 'microsoftTeamsUser' | 'unknown';
```

Comment:
I think you just need to build again to update the removal of the `kind` property.